vllm/multimodal/utils.py

```diff
@@ -23,30 +24,26 @@ def get_aiohttp_client(cls) -> aiohttp.ClientSession:
 
         return cls.aiohttp_client
 
-    @classmethod
-    async def close_aiohttp_client(cls) -> None:
-        if cls.aiohttp_client:
-            await cls.aiohttp_client.close()
-            cls.aiohttp_client = None
-
     @classmethod
     async def fetch_image(cls, image_url: str) -> Image.Image:
-        """Load image from a url or base64 encoded openai GPT4V format"""
+        """Load PIL image from a url or base64 encoded openai GPT4V format"""
 
-        # Avoid circular import
-        from vllm import __version__ as VLLM_VERSION
+        if image_url.startswith('http'):
+            # Avoid circular import
+            from vllm import __version__ as VLLM_VERSION
 
-        client = cls.get_aiohttp_client()
-        headers = {"User-Agent": f"vLLM/{VLLM_VERSION}"}
+            client = cls.get_aiohttp_client()
+            headers = {"User-Agent": f"vLLM/{VLLM_VERSION}"}
 
-        if image_url.startswith('http'):
             async with client.get(url=image_url, headers=headers) as response:
                 response.raise_for_status()
                 image_raw = await response.read()
             image = Image.open(BytesIO(image_raw))
 
         elif image_url.startswith('data:image'):
-            image = load_image_from_base64(image_url.split(',')[1])
+            loop = asyncio.get_event_loop()
+            image = await loop.run_in_executor(None, load_image_from_base64,
+                                               image_url.split(',')[1])
```

Comment:
- I don't see how this way is different from using `make_async`.
- We should set a limit of `1` to `image_url.split` so it won't keep iterating through the string.

Reply:
- Good catch - I was testing if this had any perf implication by making it async but forgot to change it back.
- That's a good point about split for sure!